packages/sanity/src/core/form/components/EditPortal.tsx

```diff
@@ -42,22 +46,24 @@ export function EditPortal(props: Props): React.ReactElement {
 
   if (type === 'dialog') {
     return (
-      <VirtualizerScrollInstanceProvider
-        scrollElement={documentScrollElement}
-        containerElement={containerElement}
-      >
-        <Dialog
-          header={header}
-          id={id || ''}
-          onClickOutside={onClose}
-          onClose={onClose}
-          width={width}
-          contentRef={setDocumentScrollElement}
-          __unstable_autoFocus={autofocus}
+      <div onDragEnter={onDragEnter}>
```

Comment:
Do we need to wrap this in a div? When I test this, the issue seems to be fixed by adding the `onDragEnter` function to the `Dialog` component. However, there might be some situations that I am not aware of where the div is needed!

Reply:
No, you are absolutely right, I didn't realised the dialog was spreading the props. thanks!